lib/timer.cpp

```diff
@@ -38,22 +31,25 @@ namespace {
     }
 }
 
+// TODO: this does not include any file context when SHOWTIME_FILE thus rendering it useless - should we include the logging with the progress logging?
```

Comment:
This would also be without context when using `-q` so tacking it with the progress won't help there and we need to add some kind of header.